librz/core/p/core_plugin_example.c

```diff
@@ -1,41 +1,93 @@
 // SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
 // SPDX-License-Identifier: LGPL-3.0-only
-#if 0
-gcc -o core_plugin_example.so -O3 -std=c99 -Wall -fPIC `pkg-config --cflags --libs rz_core` core_plugin_example.c -shared
-mkdir -p ~/.config/rizin/plugins
-mv core_plugin_example.so ~/.config/rizin/plugins
-#endif
+
```

Comment:
Awesome! Is this compiled anyhow while testing rizin? I was also wondering.. Aren't unit tests enough to showcase how to write a new command?

Reply:
yes

Reply:
Where is this compiled?

Reply:
on my machine.